arithmetization/src/test/java/net/consensys/linea/zktracer/ReplaysTests.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Consensys Software Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer;
+
+import net.consensys.linea.zktracer.testing.ToyExecutionEnvironment;
+import org.junit.jupiter.api.Test;
+
+public class ReplaysTests {
+  public static void replay(String filename) {
+    ToyExecutionEnvironment environment = ToyExecutionEnvironment.builder().build();
+    environment.replay(filename);
+  }
+
+  @Test
+  void replayOneFile() {
+    replay("/home/franklin/asdf.json");
```

Comment:
put smtg more generic ;)

Reply:
As soon as we have a fix for the tx tracing issue.